src/main/scala/org/zalando/nakadi/client/actor/KlientSupervisor.scala

```diff
@@ -50,7 +50,7 @@ class KlientSupervisor private (val endpoint: URI, val port: Int, val securedCon
 
   var listenerMap: Map[String, ActorRef] = Map()
 
-  override val supervisorStrategy = AllForOneStrategy(maxNrOfRetries = 100, withinTimeRange = 5 minutes) {
+  override val supervisorStrategy = AllForOneStrategy(maxNrOfRetries = 100, withinTimeRange = 5 minutes) {    // config?
```

Comment:
Akka is an interesting thing. I would kind of assume these to be configurable from within it, but last time I checked, they weren't. I envision there is some reason behind this, but am not sure. Will add to Config, tomorrow.